pinot-clients/pinot-java-client/src/test/java/org/apache/pinot/client/ExecutionStatsTest.java

```diff
@@ -39,9 +39,12 @@
   public void setUp()
       throws Exception {
     String json =
-        "{\"numServersQueried\":10, \"numServersResponded\":10, \"numDocsScanned\":10, \"numEntriesScannedInFilter\":10, "
-            + "\"numEntriesScannedPostFilter\":10, \"numSegmentsQueried\":10, \"numSegmentsProcessed\":10, \"numSegmentsMatched\":10, "
-            + "\"numConsumingSegmentsQueried\":10, \"minConsumingFreshnessTimeMs\":10, \"totalDocs\":10, \"numGroupsLimitReached\":true, "
+        "{\"numServersQueried\":10, \"numServersResponded\":10, \"numDocsScanned\":10, "
```

Comment:
Can we reformat this? Same for other places
I think this is caused by reformatting twice, first time 140 then 120